kensu/requests/api.py

```diff
@@ -11,33 +13,49 @@ def wrapper(*args, **kwargs):
 
         kensu = KensuProvider().instance()
 
-        result_pk = DataSourcePK(location=result.url,
-                                 physical_location_ref=kensu.default_physical_location_ref)
-        result_ds = DataSource(name=result.url, format='API',
-                               pk=result_pk)._report()
+        try:
+            fields_dict = flatten(result.json())
```

Comment:
p.s. I think there should be a separate try/catch only for json. if json parsing fails, we use some arbitrary field name, like `data/value`?

Reply:
ok, foe now we only handle the case where result is a json/dict